f5-distributed-cloud/README.md

```diff
@@ -0,0 +1,99 @@
+## Overview
+
+F5 Distributed Cloud (XC) Services provides a global cloud native platform where customers can deploy, manage, and secure their applications regardless of whether the application resides in a public cloud, in a private data center, or a colocation facility (see below). It provides a variety of both ADN and CDN services.
+
+![snapshot][1]
```

Comment:
The overview section doesn't support images. This needs to be moved to the manifest file under media

Reply:
Hello I have updated all image sizes.  Should I initiate a new pull request?

Reply:
removed